mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -144,6 +154,7 @@ def finish(self):
                     self.selected_resource.closeConnection()
                 if self.selected_resource.is_allocated:
                     self.selected_resource.release()
+                    self.selected_resource = None
```

Comment:
> Do you mean user will try again?

actually not. This whole function is called twice by htrun - once from core, and once when this class destructor is called. So we actually retry already twice. Anyway this shouldn't normally fail ever. If we really want to do retry for this we should do retry for other calls as well and perhaps more intelligent logic for it -> I wouldn't like to add any retry logic here for now. Resource is released anyway after given expiration time in remote side so it doesn't cause big harm even this fails here.